src/app/collections/Collections.js

```diff
@@ -0,0 +1,159 @@
+          </div>
+          <h1 className="text-secondary">Loading decks...</h1>
+        </div>
+      );
+    }
+
+    if (isError) {
+      return (
+        <div className="container p-4 my-5">
+          <div className="mb-5">
+            <h1 className="m-0">Flashcards for Developers</h1>
+            <p>A curated list of flashcards to boost your professional skills</p>
+          </div>
+          <div className="text-center mt-3">
+            <h1 className="text-dark">Unable to load request</h1>
+            <p>Please try again or contact us.</p>
+          </div>
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        <div className="container px-0">
+          <ReviewNavigation location={this.props.location} />
+        </div>
+        <div className="container py-4">
+          <div className="d-flex flex-column flex-lg-row justify-content-between align-items-lg-center">
+            <div className="mb-3">
+              <h1 className="m-0">{collection.name}</h1>
+              <p className="m-0">{collection.description}</p>
+            </div>
+            <div
+              className="bg-light rounded p-3 mb-2 border border-secondary d-flex align-items-center"
+              style={{ minWidth: "260px", minHeight: "90px" }}
+            >
+              <SkillProgress decks={decks} />
+            </div>
+          </div>
+          {decks.length > 0 ? (
+            <div className="row pt-1">
+              {decks.map(deck => (
+                <DeckItem
+                  deck={deck}
+                  key={deck.id}
+                  location={location}
+                  isSaved={this.isSaved(deck.id)}
+                  onToggleSave={this.onToggleSave}
+                />
+              ))}
+            </div>
+          ) : (
+            <div className="w-100 text-center my-5 pb-5" style={{ minHeight: "30vh" }}>
+              <div className="h4" style={{ opacity: 0.8 }}>
+                This collection is currently empty.
+              </div>
+              <div className="mb-4">Let us know what topics you want to see.</div>
+              <div className="row d-flex justify-content-center mt-2 mb-5">
+                <a
+                  className="text-dark d-flex align-items-center btn btn-outline-dark px-4"
+                  href={config.airtableSuggestionsUrl}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  style={{ borderRadius: "999px" }}
+                >
+                  <Octicon className="d-flex mr-2" name="plus" />
+                  <span>Suggest a deck</span>
+                </a>
+              </div>
+            </div>
+          )}
+          <div className="row mt-5">
+            <div className="col-md-10 offset-md-1 col-lg-8 offset-lg-2 mt-5">
+              <FeedbackForm />
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
```

Comment:
What do you think about adding the suggest a deck/feedback form to the bottom?
![image](https://user-images.githubusercontent.com/1075072/44961025-bb0cd600-aebe-11e8-9444-a640d5dc0726.png)

Reply:
The "Subscribe" element is already there. Let's leave off the "Suggest a deck" element on the collections pages.

Reply:
👍